.gitignore

```diff
@@ -30,3 +30,4 @@ cmake-build-debug
 #Docker
 docker/docker-compose.override.yml
 docker/.env
+*.lua
```

Comment:
Is this intended? We won't be able to add new .lua files to the git repo with this in.